spec/acceptance/generate_markdown_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'spec_helper_acceptance'
-require 'util'
-
-include PuppetStrings::Acceptance::Util
 
 describe 'Generating Markdown' do
+  before(:all) do
+    @test_module_path = sut_module_path(/Module test/)
+    @remote_tmp_path = sut_tmp_path
+  end
```

Comment:
Unless these are particularly expensive or need to be shared between examples, I would prefer these to be `let` or `before(:each)`

Reply:
I did originally use `:each` and it works fine

* They're not too expensive on Linux Docker containers, but once this tests on Windows over WinRM, they will take a while (5+ seconds per call IIRC)

* The values don't change during the lifetime of the SUT so I didn't see the point of running the same commands multiple times.

I don't have a hard opinion either way. Both approaches work, and this is faster.  That was all.